src/Faker/Provider/HtmlLorem.php

```diff
@@ -43,10 +43,16 @@ public function __construct(Generator $generator)
      * @param int $maxDepth
      * @param int $maxWidth
      *
+     * @throws \RuntimeException
```

Comment:
Throws doc doesn't need to document `RuntimeException`s or `Error`s. Code is assumed to always be able to throw those kind of things.

Reply:
Added it because it was requested (https://github.com/FakerPHP/Faker/pull/120#discussion_r546606173). You're right @GrahamCampbell - thanks!